src/main/webapp/i18n/en/exercise.json

```diff
@@ -140,6 +140,9 @@
                 },
                 "maxScore": {
                     "pattern": "The maximum score must be between 1 and 9999!"
+                },
+                "maxPenalty": {
+                    "pattern": "The maximum penalty must be between 0 and 100!"
```

Comment:
You could add that the penalty is in percent (as in the german translation).

Reply:
Added this to the string